### PR TITLE
Speedtestdotnet - Remove section on manual polling

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Speedtestdotnet - Remove the specific section on manual polling
- as information is not integration specific
- Reverts PR #28609
- Create a reusable topic under **Common tasks** section
- Reuse the topic in topics that mention manual polling

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dirrgang** [comment]
The information should then also be removed from https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/waze_travel_time.markdown I imagine.

I feel like this *should* be documented somewhere. Googling "home assistant manual polling" leads to https://www.home-assistant.io/docs/configuration/platform_options/, which is apparently being phased out?

Is this something worth documenting somewhere? Perhaps also under advanced configuration? (I wouldn't mind giving it a try)

File: source/_integrations/speedtestdotnet.markdown [hunk 1]
```diff
@@ -22,9 +22,7 @@ The Speedtest.net integration uses the [Speedtest.net](https://speedtest.net/) w
 
 Most Speedtest.net servers require TCP port 8080 outbound to function. Without this port open you may experience significant delays or no results at all. See note on their [help page](https://www.speedtest.net/help).
 
-## Manual Polling
-
-By default, a speed test will be run every hour. To customize polling, you can disable automated polling. Go to {% my integrations title="**Settings** > **Devices & Services**" %}, and select the **Speedtest** integration. On the integration entry, select the three dots. Then, select **System options** and toggle the button to disable polling. To manually trigger a polling request, call the [`homeassistant.update_entity` service](/integrations/homeassistant/#service-homeassistantupdate_entity) as needed, either manually or via automations.
```
Comment (frenck) [inline source/_integrations/speedtestdotnet.markdown:27]: Should we create an item like you did for enabling entities? And reference that instead?

Like `{% include common-tasks/enable_entities.md %}` (which is also included on the independent common tasks lists).

Comment (c0ffeeca7) [reply source/_integrations/speedtestdotnet.markdown:27]: Thank you for pointing that out! Moved the procedure into the common tasks section.

**Comment by c0ffeeca7** [comment]
> The information should then also be removed from https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/waze_travel_time.markdown I imagine.
> 
> I feel like this _should_ be documented somewhere. Googling "home assistant manual polling" leads to https://www.home-assistant.io/docs/configuration/platform_options/, which is apparently being phased out?
> 
> Is this something worth documenting somewhere? Perhaps also under advanced configuration? (I wouldn't mind giving it a try)

Thank you for pointing that out. Based on your text, I created a reusable topic in the **Common tasks** section and applied it to integrations that mention manual polling.

**Review by frenck — APPROVED** [review]
Nice! ❤️ 

../Frenck

**Comment by starkillerOG** [comment]
@c0ffeeca7 the section title for the "changing polling intervall" section is missing (in my opinion) here: https://www.home-assistant.io/common-tasks/general
It now is unclear what belongs to the enabeling entities and chaning polling intervall sections

**Comment by c0ffeeca7** [comment]
> @c0ffeeca7 the section title for the "changing polling intervall" section is missing (in my opinion) here: https://www.home-assistant.io/common-tasks/general It now is unclear what belongs to the enabeling entities and chaning polling intervall sections

Oh dear. Thank you for the catch, @starkillerOG! Let me fix this:
#28661

**Pull request merged**